rmgpy/molecule/translator.py

```diff
@@ -134,72 +134,76 @@
 }
 
 
-def toInChI(mol, backend='try-all'):
+def toInChI(mol, backend='try-all', aug_level=0):
     """
-    Convert a molecular structure to an InChI string. Uses
-    `RDKit <http://rdkit.org/>`_ to perform the conversion.
-    Perceives aromaticity.
+    Convert a molecular structure to an InChI string.
+    For aug_level=0, generates the canonical InChI.
+    For aug_level=1, appends the molecule multiplicity.
+    For aug_level=2, appends positions of unpaired and paired electrons.
```

Comment:
`aug_level` of 2 seems to also append multiplicity. 

Can you think of a usage where someone wants the aug_level 2 but not have multiplicity? Maybe it doesn't exist, but if it does, it might be useful to have aug_layer be a list or tuple.

Reply:
Though maybe this is outside the scope of the PR also

Reply:
Do you mean based on the docstring or in the code? I'm pretty sure that `aug_level=2` doesn't include multiplicity. I chose to leave `aug_level=2` as the augmented InChI as Nick had implemented. `aug_level=1` is essentially the old augmented InChI before Nick revamped it.

I think Nick's idea was that specifying unpaired electrons would be sufficient to calculate multiplicity, which was the case before we started worrying about excited states and such. I could add the multiplicity to `aug_level=2`, which I don't think would hurt anything, since it's only being used for labeling stuff right now.